README.md

```diff
@@ -10,18 +10,42 @@ By running the script before and after joining the team, you can see which repos
 
 Explanations of the various functionality is detailed below, but an expected workflow looks like:
 
+#### 1. Get the repo!
 ```bash
 # Clone the repo
 gh repo clone exercism/gh-notifications-manager
 cd gh-notifications-manager
+```
+
+#### 2. Export your current notification preferences
+```bash
+bin/manage export
+```
+
+This creates a `subscriptions.json`. 
+Back this file up!
+You'll want this copy of it later.
+
+#### 3. Tell admins you're ready
 
-bin/manage export          # Creates a record of your subscriptions
+When you've done this, tell us and we'll add you to whatever teams you're being added to.
+This will likely subscribe you to lots of repos.
 
-# Manually: Backup file
-# Manually: Join relevant teams
+#### 4. Reset your notifications
 
-bin/manage export          # Updates the record with new subscriptions
-bin/manage unsubscribe_new # Changes the state in the file to UNSUBSCRIBED for all new repos
+If your original `subscriptions.json` file contains all Exercism repos (which it seems to for lots of people) then you can simply run:
```

Comment:
I don't think this is true anymore. Maybe someone could try out the latest version of the script without a subscriptions.json file present?

Reply:
This would probably be the same as just deleting the file and starting over BTW